operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -340,6 +344,24 @@ type LocalScannerComponentSpec struct {
 	DB *DeploymentSpec `json:"db,omitempty"`
 }
 
+// LocalScannerV4ComponentSpec defines settings for the "scanner V4" component in SecuredClusters
+type LocalScannerV4ComponentSpec struct {
+	// If you do not want to deploy the Red Hat Advanced Cluster Security local Scanner V4, you can disable it here
+	// (not recommended).
```

Comment:
Maybe mention here again the relationship with `scanner.disable`? Not sure myself.

Reply:
See above.